src/dowel/csv_output.py

```diff
@@ -56,6 +54,36 @@ def record(self, data, prefix=''):
         else:
             raise ValueError('Unacceptable type.')
 
+    def _augment(self, data):
+        """Augment tabular data with new column(s)"""
+        data.reset()
+        self._log_file.close()
+
+        temp_file = self._file_name.split(".")
+        temp_file = "".join(temp_file[:-1]) + "_temp." + temp_file[-1]
```

Comment:
This logic would probably be clearer using `os.path.splitext`.

Reply:
Thanks for the tip!